dev/tasks/r/github.devdocs.yml

```diff
@@ -59,10 +61,22 @@ jobs:
         env:
           LIBARROW_BINARY: FALSE
           ARROW_R_DEV: TRUE
-        run: bash arrow/r/vignettes/script.sh
+          DEVDOCS_WINDOWS: {{ "${{contains(matrix.os, 'windows')}}" }}
+        run: |
+          if [ $DEVDOCS_WINDOWS == "true" ]; then
+            # This starts a special mingw64 Git Bash shell
+            $RTOOLS40_HOME/msys2_shell.cmd -here -mingw64 -no-start -defterm -full-path arrow/r/vignettes/script.sh
+          else 
+            bash arrow/r/vignettes/script.sh
+          fi
         shell: bash
       - name: Ensure that the Arrow package is loadable and we have the correct one
+        env:
+          DEVDOCS_WINDOWS: {{ "${{contains(matrix.os, 'windows')}}" }}
         run: |
+          if [ $DEVDOCS_WINDOWS == "true" ]; then
+            export PATH=/c/R/bin:$PATH
+          fi
```

Comment:
Oh... I didn't connect those dots 🤦‍♂️. I added the above because I thought R was missing from the path (but I don't know that for sure) and I added the below because R was on the path but formatted in Windows style (something I reproduced locally). I'll try adding the cygwin conversion in the `.bash_profile` and see if it runs smoothly.

Reply:
Okay yeah, I just did the cygwin conversion there. Also needed to wrap the path in quotes because of the spaces in Windows paths.